modules/oe_theme_content_news/oe_theme_content_news.post_update.php

```diff
@@ -76,3 +76,25 @@ function oe_theme_content_news_post_update_00005(): void {
     }
   }
 }
+
+/**
+ * Override news default view display.
+ */
+function oe_theme_content_news_post_update_00006(): void {
```

Comment:
We don't need oe_theme_content_news_post_update_00006(). Please update core.entity_view_display.node.oe_news.default in the 00005_override_view_displays folder.

Reply:
Updated

Reply:
We need to enable the module in an update hook.